scripts/o3de/tests/test_engine_template.py

```diff
@@ -14,7 +14,7 @@
 from o3de import engine_template
 from unittest.mock import patch
 
-CPP_LICENSE_TEXT = """// {BEGIN_LICENSE}
+CPP_LICENSE_TEXT_TEMPLATE = """// {BEGIN_LICENSE}
```

Comment:
I wonder if it would be easier to define `CPP_LICENSE_TEXT_TEMPLATE` in terms of `CPP_LICENSE_TEXT`.

Example: `CPP_LICENSE_TEXT_TEMPLATE = "// {BEGIN_LICENSE}\n" + CPP_LICENSE_TEXT + "\n//{END_LICENSE}"`

Reply:
Yeah, I can do that.